docs/specs/markdown.yml

```diff
@@ -297,6 +297,49 @@ outputs:
   docs/a.json: |
     { "conceptual": "<div></div><p>body</p>" }
 ---
+# table with styles is allowed
+inputs:
+  docfx.yml:
+  docs/a.md: |
```

Comment:
consider put the test in a `HtmlUtilityTest.HtmlStripTags` UT

Reply:
it's the case like spec test `do not allow custom js, styles and css`, so keep it in spec test may be ok?

Reply:
Yeah, the table is fine, just those other tests